packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -145,9 +134,10 @@ export class GroupedListBase extends BaseComponent<IGroupedListProps, IGroupedLi
       selectionMode,
       selection,
       viewport,
-      onShouldVirtualize
+      onShouldVirtualize,
+      groups
     } = this.props;
-
```

Comment:
It seems like it would be a lot less energy to simply pass the original `groups` array to all group renderers. More powerful for consumers and less work within `GroupedListBase`.